$MirOS: ports/devel/automake/1.9/patches/patch-lib_am_mans_am,v 1.2 2005/06/13 20:10:43 tg Exp $
--- lib/am/mans.am.orig	Sat May 14 20:20:44 2005
+++ lib/am/mans.am	Sat Apr  5 17:47:32 2008
@@ -26,10 +26,11 @@ man%SECTION%dir = $(mandir)/man%SECTION%
 ## is hard coded.
 
 .PHONY: install-man
-?INSTALL-MAN?install-data-am: install-man
+?INSTALL-MAN?INSTALL_MAN_AM = install-man
+?INSTALL-MAN?install-data-am: $(INSTALL_MAN_AM)
 ?INSTALL-MAN?am__installdirs += "$(DESTDIR)$(man%SECTION%dir)"
 .PHONY install-man: install-man%SECTION%
-install-man%SECTION%: $(man%SECTION%_MANS) $(man_MANS)
+install-man%SECTION%:
 	@$(NORMAL_INSTALL)
 	test -z "$(man%SECTION%dir)" || $(mkdir_p) "$(DESTDIR)$(man%SECTION%dir)"
 	@list='$(man%SECTION%_MANS) $(dist_man%SECTION%_MANS) $(nodist_man%SECTION%_MANS)'; \
